### PR TITLE
Add Coveralls

### PR DESCRIPTION
**Pull request by mortaliorchard**
@rantonmattei

### CODE REVIEW

**Comment by rantonmattei** [comment]
R+

File: sample.py [hunk 1]
```diff
@@ -6,8 +6,8 @@
 
 app = Flask(__name__)
 flask_logger.setup(
-    app, 'https://logglyurl', 'dev', 'logger_name', logging.INFO,
-    'service_name', '1.0.0')
+    app, 'http://logs-01.loggly.com/bulk/77c54e3f-33b7-4f8b-a2a7-dbaef3414348/tag/ows1/', 'dev', 'michael_ortali', logging.INFO,
```
Comment (azinger-theorchard) [inline sample.py]: Is this our real loggly key?  This repo is open to the public.

Comment (rantonmattei) [inline sample.py]: uh oh, yes. You need to remove that.

**Comment by jessmchung** [comment]
@mortaliorchard Can you rebase this?

**Comment by rantonmattei** [comment]
poke @mortaliorchard @jessmchung

**Pull request closed without merging**